package.json

```diff
@@ -12,24 +13,22 @@
     "spago": "spago",
     "start": "vite",
     "start:spago": "spago build --watch",
-    "transform-require": "node scripts/transform-require.js",
```

Comment:
:question: The execution of `scripts/transform-require.js` is being removed in this PR.

* Why is this step not necessary anymore? Still the bundled output is used because of the HTML, right? But why was this script there before?
* If the script can be removed, the folder `scripts` should be deleted in this PR